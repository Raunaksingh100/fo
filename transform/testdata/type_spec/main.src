package main

type A[T] T
type B[T] []T
type C[T] [3]T
type D[T] map[T]T
type E[T] map[T]string
type F[T] map[string]T
type G[T] *T
type H[T] ***T
type I[T] chan T
type J[T] []map[T]chan T
type K[T, U] map[T]U
type L[T] func(T)
type M[T, U] func(T) U

func main() {

}
