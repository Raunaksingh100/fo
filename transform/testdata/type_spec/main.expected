package main

type A interface{}
type B interface{}
type C interface{}
type D interface{}
type E interface{}
type F interface{}
type G interface{}
type H interface{}
type I interface{}
type J interface{}
type K interface{}
type L interface{}
type M interface{}

func main() {

}
